<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudConfPage</class>
 <widget class="QWidget" name="AudConfPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>审计配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>审计日志保存位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="log_filelineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>当日志增长到      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_log_filelineEdit">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>MBs时，</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="max_log_file_action_comboBox">
          <item>
           <property name="text">
            <string>IGNORE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYSLOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ROTATE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>KEEP_LOGS</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>     保存最新的     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="num_logslineEdit">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string> 份日志文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>循环日志并保存所有</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>当磁盘剩余空间达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="admin_space_left_lineEdit">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>MBs时，</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="admin_space_left_action_comboBox">
          <item>
           <property name="text">
            <string>IGNORE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYSLOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EMAIL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SINGLEHALT</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>142</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>当磁盘剩余空间达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="space_left_lineEdit">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>MBs时，</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="space_left_action_comboBox">
          <item>
           <property name="text">
            <string>IGNORE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYSLOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EMAIL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SINGLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HALT</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>142</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_20">
        <item row="0" column="2">
         <widget class="QComboBox" name="disk_full_action_comboBox">
          <item>
           <property name="text">
            <string>IGNORE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYSLOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SINGLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HALT</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>142</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="disk_error_action_comboBox">
          <item>
           <property name="text">
            <string>IGNORE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYSLOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SINGLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HALT</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>当磁盘满时</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>当磁盘输入错误时</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_15">
        <property name="title">
         <string>报警方式</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QLineEdit" name="mail_lineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="msg_lineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>收件人(多人用逗号隔开):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>短信(多人用逗号隔开):</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="apl_cfg_Button">
          <property name="text">
           <string>应用</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="restartAduButton">
        <property name="text">
         <string>重启审计服务</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
